index.bs

```diff
@@ -1321,6 +1590,57 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 	</div><!-- algorithm -->
 
+	<div class="algorithm" data-algorithm="write-blobs-to-clipboard">
+	<h3 id="to-write-blobs-to-clipboard"><dfn>write blobs and option to the clipboard</dfn></h3>
+
+		: Input
+		:: |items|, a [=sequence=]&lt;{{Blob}}&gt; list of {{Blob}}s to write
+		:: |option|, a [=ClipboardItem/clipboard item=]'s [=presentation style=]
+
+		: Output
+		:: None
+
+		1. If the |items| list is not empty, then
+
+			1. For each |item| in |items|:
+
+				1. If data type is <i>text/plain</i>, then
```

Comment:
What is "data type"? Maybe you mean |item|'s {{Blob/type}}? If so, that's a string; how do you want to compare it to text/plain? Do charset or other parameters matter? What about capitalization and other differences that might arise from parsing?